README.md

```diff
@@ -92,7 +92,37 @@ A wildcard stands for all the non-specified HTTP methods.
 
 ##### Key: `<cache-header-name>` 
 
-A http cache header name (`Cache-control`, `Expires`, etc.).
+A http cache header name (`cache-control`, `expires`, `etag` etc.).
+
+###### ETags
+
+For Etags you can specify a custom generator in the configuration:
+
+```
+'etag' => array(
+    'override' => true,
+    'generator' => 'Your\Own\EtagGenerator'
+)
+```
+
+Example generator:
+
```

Comment:
I would just reference DefaultEtagGenerator class as an example. Avoiding to have to change the example when it changes.